src/app/bisoncoin/src/main.ts

```diff
@@ -1,10 +1,24 @@
-import Vue from 'vue'
+import Vue from 'vue';
+import VueRouter from 'vue-router';
 import App from './App.vue'
 import store from './store'
 
-Vue.config.productionTip = false
+Vue.use(VueRouter);
+
+import HomePage from "./pages/HomePage.vue"
```

Comment:
Can we move these imports to the top or does it need to be underneath Vue.use(VueRouter)?

Reply:
They can be at the top! Fixed